src/api/api.ts

```diff
@@ -0,0 +1,24 @@
+import { Comment, CommentData } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+import { client } from '../utils/fetchClient';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
+};
+
+export const getPosts = (userId: number) => {
+  return client.get<Post[]>(`/posts?userId=${userId}`);
+};
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const deleteComment = (commentId: number) => {
+  return client.delete(`/comments/${commentId}`);
+};
```

Comment:
The `deleteComment` function lacks error handling. Consider adding a `.catch()` block to handle potential errors that may occur during the API call. This will help in providing feedback to the user if the deletion fails.